toolchains/osx.toolchain.cmake

```diff
@@ -39,17 +39,40 @@ set( TNUN_os_suffix OSX )
 
 set( TNUN_cpu_archs default )
 
+set( TNUN_ABI x64 )
```

Comment:
hm yes...universal build don't play well with this scheme...
so let's leave it like that for now...and later we can then add separate x86 and x86+x64 'unibuild' abis if needed...